PizzaPyWebApp/templates/partials/landing/about_us.html

```diff
@@ -0,0 +1,75 @@
+<div class="container px-4 py-12 mx-auto">
+    <div class="flex flex-col items-center gap-12 md:flex-row">
```

Comment:
Change `<div class="flex flex-col items-center gap-12 md:flex-row">` to `<div class="flex flex-col items-center gap-20 md:flex-row">` to match the Figma design's spacing.